packages/carbon-react/package.json

```diff
@@ -92,6 +92,8 @@
   },
   "sideEffects": [
     "es/feature-flags.js",
-    "lib/feature-flags.js"
+    "lib/feature-flags.js",
+    "es/prefix.js",
```

Comment:
Out of curiosity, why does these files (really all of them) have to be listed as sideEffects?

Reply:
I'm also curious. My guess is since it's pulled from `carbon-components` these should never be pruned when tree shaking?

Reply:
@abbeyhrt @tay1orjones basically these modules are not "pure" because they cause side effects in other modules (feature flags, components).

Bundlers like webpack/rollup will assume modules are pure by default if `sideEffects: false` is listed. This means that if exports from these entrypoints are used that the bundler will actually drop the file from the final output. This is not what we want because we always want these files to be included and their side effects run.

Annotating these files as having side effects ensures that they are included in the final bundle even though we aren't using exports from these files.

Hope this helps!